packages/components/src/date-time/date.js

```diff
@@ -36,7 +36,7 @@ class DatePicker extends Component {
 	}
 
 	render() {
-		const { currentDate } = this.props;
+		const { currentDate, isInvalidDay } = this.props;
```

Comment:
Probably not totally required but should we provide a default prop for this of `false` just to be extra explicit?

Reply:
Thanks @timmyc !

I could set a default prop or just `{ isInvalidDay = false } = this.props`, but I think its best to pass along an `undefined` value to react-dates's own default prop for `isOutsideRange`:

https://github.com/airbnb/react-dates/blob/d572bea9029739089bd200026be3aad7684a8b02/src/components/DayPickerSingleDateController.jsx#L100

The reason is because React handles an `undefined` prop differently than `null` or possibly `false` (https://reactjs.org/docs/react-component.html#defaultprops).